pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/autogen/requests/GetCreditCards.java

```diff
@@ -0,0 +1,71 @@
+// DO NOT EDIT: File is generated by code generator.
+package jp.pokepay.pokepaylib.BankAPI.autogen.requests;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import jp.pokepay.pokepaylib.BankAPI.BankRequest;
+import jp.pokepay.pokepaylib.BankAPI.BankRequestError;
+import jp.pokepay.pokepaylib.ProcessingError;
+import jp.pokepay.pokepaylib.Request;
+import jp.pokepay.pokepaylib.BankAPI.autogen.responses.*;
+
+public class GetCreditCards extends BankRequest {
+    private String userId;
+    private String before;
+    private String after;
+    private Integer perPage;
+    private String organizationCode;
+
+    public GetCreditCards(String userId, String organizationCode) {
+        this.userId = userId;
+        this.organizationCode = organizationCode;
+    }
+
+    public GetCreditCards before(String before) {
```

Comment:
Why is the naming for these optional setter functions different if they are autogenerated?  It was `setCardholderNameSpecified` in CreateCreditCard.  Shouldn't this be `setBefore`?

Reply:
Generally, I think that a builder pattern (which may not strictly be a builder pattern) is not a set, but only a name.

Reply:
For example
```
new GetCreditCards(userId, organizationCode)
  .after("xxx")
  .perPage(20);
```

Reply:
Then, shouldn't we change `setCardholderNameSpecified` to just `cardholderNameSpecified` or `isCardholderNameSpecified`?

Reply:
Oh, sorry.
This was not a response to your comment.
It seems to be a bug in the generator that malfunctions when the prefix of the openapi parameter name is "is".

Reply:
fixed it